css/style.css

```diff
@@ -829,3 +829,194 @@ ul {
 }
 
 /* 3rd subpage */
+
+
+body {
+  font-family: 'Roboto-medium', sans-serif;
+  margin: 0;
+  padding: 0;
+}
+
```

Comment:
Many empty lines. Please have a look at the rest of the code, I won't write any more comments regarding these empty lines